src/chrome/content/rules/MSN.xml

```diff
@@ -202,34 +196,20 @@
 	<target host="byfiles.storage.msn.com" />
 
 	<target host="udc.msn.com" />
-	<target host="catalog.video.msn.com" />
 	<target host="*.ssl.catalog.video.msn.com" />
-	<!--target host="www.msn.com" /-->
 
 	<!--	Complications:
 				-->
 	<target host="h.msn.com" />
 	<target host="imgs1-video.msn.com" />
-	<target host="c.jp.msn.com" />
-
-	<target host="c.sg.msn.com" />
```

Comment:
I removed the hosts because Travis was failing when they were still in there. But looking back at what I removed, it looks like I removed some subdomains that were fine. So I put those back in. There are some others that you and Travis disagree on though (see https://travis-ci.org/EFForg/https-everywhere/jobs/224183757)
Error messages for subdomains in the above link:
* bedrockformsemea.msn.com
  * `ERROR src/chrome/content/rules/MSN.xml: Fetch error: http://bedrockformsemea.msn.com/ => https://bedrockformsemea.msn.com/: (6, 'Could not resolve host: bedrockformsemea.msn.com')`
* bedrockformsemeanext.msn.com
  * `ERROR src/chrome/content/rules/MSN.xml: Fetch error: http://bedrockformsemeanext.msn.com/ => https://bedrockformsemeanext.msn.com/: (6, 'Could not resolve host: bedrockformsemeanext.msn.com')`
* c.sg.msn.com
  * `ERROR src/chrome/content/rules/MSN.xml: Non-2xx HTTP code: http://c.sg.msn.com/ (200) => https://c.msn.co.jp/ (403)`

I put these back:
* bedrockbluenext.msn.com
* bedrockformsbluenext.msn.com
* bedrockformskaw.msn.com
* c.jp.msn.com